tests-ng/test_users.py

```diff
@@ -30,3 +31,27 @@ def test_no_extra_home_directories(expected_users):
     entries = os.listdir("/home")
     unexpected = [e for e in entries if e not in expected_users]
     assert not unexpected, f"Unexpected entries in /home: {entries}"
+
+
+@pytest.mark.booted
+@pytest.mark.root
+def test_users_sudo_capability(expected_users, user: User):
+    for entry in pwd.getpwall():
+        if (entry.pw_name == "root") or (entry.pw_name in expected_users):
+            assert user.is_user_sudo(
+                entry.pw_name
+            ), f"User: {entry.pw_name} doesn't have sudo permission"
+        else:
+            assert (
+                user.is_user_sudo(entry.pw_name) == False
+            ), f"System User {entry.pw_name} has sudo permission"
+
+
+@pytest.mark.booted
+@pytest.mark.root
+def test_available_users(expected_users, regular_user_uid_range):
+    for entry in pwd.getpwall():
+        if entry.pw_uid in regular_user_uid_range:
```

Comment:
I feel like getting this list should be in a plugin, too many details for the test

maybe have some `get_users_in_regular_uid_range()` function, and then you can call this here and assert on the expected list returned?